annotations/tekton-annotations/src/main/java/io/dekorate/tekton/annotation/TektonApplication.java

```diff
@@ -108,11 +108,11 @@
   String sourceWorkspaceClaim() default "";
 
   /*
-   * The size requirement of the generated PVC
+   * The size requirement of the generated PVC in gigs.
```

Comment:
Maybe it should be megabytes instead?

Reply:
This needs to be configurable, but right now I need a quick fix.